### PR TITLE
Add alert color

### PR DESCRIPTION
**Pull request by Souldjinn**
Modify the color value to show up red when prs have been opened 
 for longer than a day.
Added a TODO for allowing configuration of this value. It could
  be worth putting it in a config file and loading it here based
  on preference.

Added some tests
Would be willing to not merge this till TODO is complete.

### CODE REVIEW

**Comment by davidwrpayne** [comment]
travis is failing

**Comment by davidwrpayne** [comment]
It would be really great if we moved some of this to a configurable file and allowed perhaps more than 2 colors as well for different lengths of time that a pr was open for.

**Comment by brentdrich** [comment]
Make this configurable with colors using the config (preferred) or update the README.

**Comment by brentdrich** [comment]
Existing config system please - https://github.com/brentdrich/prmonitor/blob/master/cmd/prmonitor/serve.go#L25

**Comment by brentdrich** [comment]
@Souldjinn

File: README.md [hunk 1]
```diff
@@ -62,3 +71,9 @@ vendor directory (so master doesn't have to) and pushes it out to heroku automat
     ```
 
 The dashboard can be run on a free hobby dyno and refreshes once every 24 hours.
+
+## Customized ranges and colors
+    # Add customizations to the CONFIG env value as shown in the example for settingup heroku.
```
Comment (brentdrich) [inline README.md]: Isn't this H1 in Markdown land?

Comment (brentdrich) [inline README.md]: Disregard, this is interpreted as a code block.

**Comment by brentdrich** [comment]
screen shot 2017-07-06 at 9 27 20 am
Looks good! I think the UX might be improved by graying out closed PRs - we still want to see how long they took but we don't want to draw attention to them with bright colors.

**Comment by Souldjinn** [comment]
prmonitor (with default customizations) now looks like this: 
screen shot 2017-07-17 at 9 50 23 pm

File: prmonitor.go [hunk 2]
```diff
@@ -309,6 +338,19 @@ func Display(in <-chan SummarizedPullRequest, w io.Writer, now time.Time, sortBy
 	return out
 }
 
+func getColor(config Config, openedFor float64, state string) string {
+	customs := config.Customization
+	if state == "closed" {
```
Comment (brentdrich) [inline prmonitor.go]: Could be better with constants, maybe.

**Comment by brentdrich** [comment]
Looks good to me.

**Pull request merged**